Doc/library/stdtypes.rst

```diff
@@ -3600,7 +3600,7 @@ copying.
          Previous versions compared the raw memory disregarding the item format
          and the logical array structure.
 
-   .. method:: tobytes()
+   .. method:: tobytes(order=None)
```

Comment:
Should you write it as `order='C'`?

Reply:
Is that common practice? I'd assume that `order='C'` means that a function raises a `TypeError` with a `None` argument.  I supported `None` because NumPy does, too.

OTOH NumPy also documents the function with `order='C'`, but I assumed that was an oversight or a historical quirk.

Reply:
Ah, I didn't realize that passing `None` explicitly was supported.